engine-cdi/pom.xml

```diff
@@ -155,6 +160,10 @@
       <artifactId>jul-to-slf4j</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
```

Comment:
Shouldn't this be in scope test? Or is this already controlled by dependency management?

Reply:
I believe this is already defined as a test scoped dependency, so I decided to use it as defined in dependency management.

Reply:
It is defined in the parent as a test-scoped dependency: https://github.com/camunda/camunda-bpm-platform/blob/7eb47150f96696ac78d9ce9682e306615af80061/parent/pom.xml#L249